lessons/Algebra2/periodic1-exploring-rides/langs/en-us/index.adoc

```diff
@@ -324,15 +397,45 @@ As with degrees, switching our unit-clock graph from hours to radians doesn't ch
 |===
 }
 
-_These are conventions that people have agreed upon over time to make it easy to collaborate. If somebody wanted to make a clock with the numbers written backwards and have the hands move the other way, it would be possible to learn how to tell time using their clock... but it likely would be more confusing than helpful for most people if we hung a clock like that in our school._
+These are conventions that people have agreed upon over time to make it easy to collaborate. It's like driving on the right side of the road v. the left: it doesn't matter what we choose, as long as everyone makes the same choice!
 
 @slidebreak
 
+We could make a clock with the numbers written _backwards_ and have the hands move the other way! And as long as everyone uses the same clock, we can still tell time._
+
 @lesson-instruction{
-Complete the table on @printable-exercise{converting-angles.adoc} with your partner, following the prompts in #1 and #2.
+- Complete the first question on @printable-exercise{converting-angles.adoc}
 }
 
+@slidebreak
+
+To plot these relationships when the input is @vocab{radians}, instead of hours, we'll use the @math{sine} and @math{cosine} functions. In Pyret (and most calculators) these function names are abbreviated as `sin` and `cos`.
```

Comment:
@schanzer I'm pretty sure this is the very first mention of **sine** and **cosine**.  Any other context we can reasonably provide at this time?

Reply:
Added some context, including some fun history